lang/en.json

```diff
@@ -33,33 +33,33 @@
           },
           "characteristics": {
             "label": "Characteristics",
-            "mgt": {
+            "M": {
```

Comment:
I think it's a lot cleaner to have internal keys be self-explanatory. We can use the full  labels, `might`, `agility`, etc., for the internal keys and leave the human-readable labels as 1 character.

Reply:
Sounds good will adjust